### PR TITLE
Add Leslie Hawthorn podcast interview blogpost

### PR DESCRIPTION
**Pull request by ElenaFdR**
null

-----
[View rendered _posts/2022-09-31-how-to-build-strong-community.md](https://github.com/publiccodenet/blog/blob/LinkedIn-post-2/_posts/2022-09-31-how-to-build-strong-community.md)

### CODE REVIEW

File: _posts/2022-09-31-how-to-build-strong-community.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+author: Kehinde Bademosi
+type: blogpost
+excerpt: Public Code Promising Practices in conversation with Leslie Hawthorn from Red Hat
+categories:
+  - Community call
+---
+
+# How to build a strong community for developing robust and valuable software
+
+![People sitting in armchairs around a coffee table writing on cards](../assets/Storytelling Workshop Public Code_Amsterdam.jpg)
```
Comment (Ainali) [inline _posts/2022-09-31-how-to-build-strong-community.md:13]:
```suggestion
![People sitting in armchairs around a coffee table writing on cards]({{site.url}}/assets/Storytelling Workshop Public Code_Amsterdam.jpg)
```

File: _posts/2022-09-31-how-to-build-strong-community.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Because it turns out if you're producing software and you know what someone expects of you, and you're trying to collaborate with other human beings, and you're trying to maintain software systems guess what? What is the thing that no one has time for?
+
+Understanding what everyone else needs and wants and how to ensure they all get there.
+
+Imagine we start small, are all friends, and know how to help each other get where we need to go at first. The more work we do and the more impact we have, the easier it is for that kind of automatic social cohesion and glue to crumble, and this is why community management and thinking about how you're going to manage your community from the very start is such an essential part of the long-term success of any project.
+
+----------------------------------------------------------------------------------------------------
+
+![Leslie Hawthorn](assets/leslie-hawthorn.jpg)
```
Comment (Ainali) [inline _posts/2022-09-31-how-to-build-strong-community.md:125]:
```suggestion
![Leslie Hawthorn]({{site.url}}/assets/leslie-hawthorn.jpg)
```

**Comment by Ainali** [comment]
Which publishing date do we want? The frontmatter and the filename should have the same date, but one is backdated to July and the other one a non-existing one.

**Comment by ElenaFdR** [comment]
@Ainali I've fixed the dates in both places now.

**Pull request merged**